### ISSUE TITLE
Video Clipping Question

### ISSUE THREAD

**Issue by DachengLi1** [PKU-YuanGroup/Open-Sora-Plan#267]
Thank you so much for the report on data and open-sourcing the annotation! I am trying to play around with the data and have two questions that hope can get your kind response. From the report:
> According to the scene transformation and clipping script provided by [Panda70M](https://github.com/snap-research/Panda-70M/blob/main/splitting/README.md), we have divided these videos into approximately 434,000 video clips. In fact, based on our clipping results, 99% of the videos obtained from these online sources are found to contain single scenes.

(1) Does this imply Panda70M incorrectly splits the videos into clips, although they are single scenes?
(2) From the data you open-source, I found they are all 65 frames. If I understand correctly, Panda70M will not result in videos with the same length. Looking from the open-sora-dataset github, I think videos are actually splitted by [no_transition.py](https://github.com/PKU-YuanGroup/Open-Sora-Dataset/blob/main/split/no_transition.py), instead of Panda70M?

Thank you so much for your potential answer!

**Comment by LinB203** [comment]
We looked at the basics of the dataset before applying the clip script on a large scale. We randomly used a few hundred videos, and we found that the results cut by the clipping tool were basically the start and end frames of the original video, this is because the dataset is almost all a single continuous scene. However applying the clipping tool significantly increases the dataset processing time, so we sliced directly by 65 frames.

By the way, our dataset has this property is because we all came from high-quality footage, and they were supposed to be a single continuous shot of footage.
